config.json

```diff
@@ -65,6 +65,19 @@
         "format": "javascript/module",
         "filter": "isCategorySize"
       }]
+    },
+    "web": {
+      "transformGroup": "js",
```

Comment:
Here I wasn't sure if we'd want to use "js" or "web" as the transform group. I chose "js" since we were already using that in the JS transform, but I'm open to changing that!

Reply:
@mtblanton "js" is correct here. "web" would get you kebab-case.